ui/app/components/mount-backend-form.hbs

```diff
@@ -39,12 +39,17 @@
       <FormFieldGroups @model={{@mountModel}} @renderGroup="Method Options">
         <:identityTokenKey>
           <SearchSelectWithModal
-            @id="identityTokenKey"
+            @id="key"
             @fallbackComponent="input-search"
+            @inputValue={{@mountModel.config.identityTokenKey}}
             @onChange={{this.handleIdentityTokenKeyChange}}
             @models={{array "oidc/key"}}
             @selectLimit="1"
             @returnString={{true}}
+            @modalFormTemplate="modal-form/oidc-key-template"
+            @placeholder="Search for an existing oidc/key, or type a new key name to create it."
+            @fallbackComponentPlaceholder="key name"
```

Comment:
nit: I think placeholder text is typically capitalized and more of an action? 
<img width="513" alt="Screenshot 2024-07-17 at 4 17 58 PM" src="https://github.com/user-attachments/assets/587caa42-db49-4df9-bce2-85f1ae23f102">
```suggestion
            @fallbackComponentPlaceholder="Input a key name"
```